### DISCUSSION TITLE
Users lost permission to select Custom Fields

### DISCUSSION THREAD

**Discussion by arlenjenks** [netbox-community/netbox#15176] [Q&A] [answered]
I'm running 3.6.6, we just upgraded to it recently. I was contacted by a co-worker this morning who I have setup as a user, it seems he's lost permission to select some custom fields I created a while back. Does anyone know if something was introduced in the 3.6.6 release that might have affected permissions? I've tried adding some permissions to his user but not having any luck, I'm not sure what permissions I would need to target that he doesn't already have.

**Comment by bogdancordos** [comment]
What version were you running before upgrading to 3.6.6? Can you replicate the steps your user is taking and share them with us, and is this working for you as an admin? You can create a dummy admin user and see if that works so you can find if this is a permission issue or something else.

**Reply by arlenjenks** [reply]
I'm not sure what version we were on before 3.6.6, someone else did the upgrade. The user navigates to Sites/*his site*/Prefixes, then selects his ip range, then drills down to the IP and selects Edit. It works for me as an admin just fine, I created a dummy user and assigned it to the same "Tech" group I have this guy in and I'm having the same issue with the dummy user, so I pretty sure it's a permissions issue. I attached an image of the error message I'm getting.

![0D156F91-68C6-4F00-A922-3CDC5A1E9D70](https://github.com/netbox-community/netbox/assets/54822460/122290af-efe5-4c96-a0c4-eb04c5e83fcd)

**Reply by bogdancordos** [reply]
What is the "Content Type" and the "Type" of the CF in question?

**Reply by arlenjenks** [reply]
Content Type is: "IPAM >IP Address"
Type is: "Selection"

Not sure why an "Antenna Password" would have a Content Type of IPAM>IP Address. Not sure if maybe this got changed in the update.

**Reply by arlenjenks** [reply, marked as answer]
I created a new permission and assigned the user to it and now it's working.
